datasets/kwaliteitsmonitor/wonenLigplaats/v1.0.0.json

```diff
@@ -16,11 +16,15 @@
     "properties": {
       "id": {
         "type": "integer",
-        "description": "Unieke id van de ligplaats"
+        "description": "Id"
       },
       "schema": {
         "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
       },
+      "bkKwaliteitsmonitorWonenLig": {
```

Comment:
Je herhaalt hier de datasetnaam en tabelnaam in het veld. Dat is niet de bedoeling. "kwaliteitsmonitor/wonenLigplaats/key" is duidelijk. Ik weet niet waar bk voor staat maar als je de rest weglaat kan je dat vol uit schrijven.